lib/llvm-backend/Cargo.toml

```diff
@@ -19,13 +19,13 @@ byteorder = "1"
 
 [target.'cfg(target_arch = "x86_64")'.dependencies.inkwell]
 git = "https://github.com/TheDan64/inkwell"
-rev = "af4cf4efbb27cdea8a54175ffc18ffd91964618c"
+rev = "5610ee7e1fea4f177d1cd70516a988df286975d8"
 default-features = false
 features = ["llvm8-0", "target-x86"]
 
 [target.'cfg(target_arch = "aarch64")'.dependencies.inkwell]
 git = "https://github.com/TheDan64/inkwell"
-rev = "af4cf4efbb27cdea8a54175ffc18ffd91964618c"
+rev = "5610ee7e1fea4f177d1cd70516a988df286975d8"
```

Comment:
You can use the `inkwell` on crates.io now, it should be the same as master on this commit and that sets us up to be able to publish the llvm-backend next release!

Reply:
AWWW YEAAHH!!